Description: Disabled the copy of the maven-shared-utils source files
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/maven-surefire-common/pom.xml
+++ b/maven-surefire-common/pom.xml
@@ -160,25 +160,6 @@
         </executions>
       </plugin>
       <plugin>
-        <!-- Remove in 3.0 -->
-        <artifactId>maven-dependency-plugin</artifactId>
-        <executions>
-          <execution>
-            <id>shared-logging-generated-sources</id>
-            <phase>generate-sources</phase>
-            <goals>
-              <goal>unpack</goal>
-            </goals>
-            <configuration>
-              <outputDirectory>${project.build.directory}/generated-sources</outputDirectory>
-              <overWriteIfNewer>false</overWriteIfNewer>
-              <artifact>org.apache.maven.shared:maven-shared-utils:3.1.0:jar:sources</artifact>
-              <includes>org/apache/maven/shared/utils/logging/*.java</includes>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
-      <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
           <includes>
--- a/maven-surefire-report-plugin/pom.xml
+++ b/maven-surefire-report-plugin/pom.xml
@@ -154,24 +154,6 @@
         </executions>
       </plugin>
       <plugin>
-        <artifactId>maven-dependency-plugin</artifactId>
-        <executions>
-          <execution>
-            <id>shared-logging-generated-sources</id>
-            <phase>generate-sources</phase>
-            <goals>
-              <goal>unpack</goal>
-            </goals>
-            <configuration>
-              <outputDirectory>${project.build.directory}/generated-sources/dependency</outputDirectory>
-              <overWriteIfNewer>false</overWriteIfNewer>
-              <artifact>org.apache.maven.shared:maven-shared-utils:3.1.0:jar:sources</artifact>
-              <includes>org/apache/maven/shared/utils/logging/*.java</includes>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
-      <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
           <useSystemClassLoader>true</useSystemClassLoader>
